amgut/webserver.py

```diff
@@ -86,6 +88,10 @@ def __init__(self):
             (r"/authed/open-humans/", OpenHumansHandler),
             (r"/authed/connect/open-humans/", OpenHumansLoginHandler),
             (r"/authed/download/(.*)", DownloadHandler),
+            (r"/authed/single/", SingleSampleHandler),
+            (r"/authed/multiple/", TaxaHandler),
```

Comment:
It isn't clear from looking at the names how `TaxaHandler` relates to `multiple`